client/driver/docker.go

```diff
@@ -470,63 +440,117 @@ func (d *DockerDriver) Start(ctx *ExecContext, task *structs.Task) (DriverHandle
 
 	// Download the image
 	if dockerImage == nil {
-		pullOptions := docker.PullImageOptions{
-			Repository: repo,
-			Tag:        tag,
+		if driverConfig.LoadImage != "" {
+			return d.loadImage(driverConfig, client, taskDir)
+		} else if driverConfig.ImageName != "" {
+			return d.pullImage(driverConfig, client, repo, tag)
+		} else {
+			return fmt.Errorf("either image name or load image has to be specified")
```

Comment:
Probably want to put a check in the validate that if load image is specified so is image name